app/models/local_delivery_unit.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# This is the new (2021) LDU - Local Delivery Unit.
+# The old (incorrectly named) one will be removed in February 2021 after the PDU/LDU go live
+class LocalDeliveryUnit < ApplicationRecord
+  VALID_COUNTRIES = ['England', 'Wales'].freeze
+
+  validates :country, inclusion: { in: VALID_COUNTRIES }
+  validates :code, presence: true, uniqueness: true, format: { with: Team::TEAM_LDU_CODE_REGEX }
+  validates :name, presence: true
+  validates :email, presence: true, 'valid_email_2/email': true
```

Comment:
It could be nice to implement auto-trimming of email addresses here. In fact, a simple gem such as [`auto_strip_attributes`](https://github.com/holli/auto_strip_attributes) might be a nice way to abstract it – to auto-trim the name and code, too.

This could make a nice re-usable pattern for MO-266 to follow.